apps/condo/domains/meter/utils/serverSchema/meterReadingDataMapper.js

```diff
@@ -1,14 +1,15 @@
 const meterReadingDataMapper = ({ row = {}, constants = {} }) => ({
     header: {
-        ru: ['Дата снятия', 'Адрес', 'Квартира', 'Услуга', 'Номер прибора', 'Место', 'Показание по тарифу №1', 'Показание по тарифу №2',
+        ru: ['Дата снятия', 'Адрес', 'Квартира', 'Лицевой счет', 'Услуга', 'Номер прибора', 'Место', 'Показание по тарифу №1', 'Показание по тарифу №2',
             'Показание по тарифу №3', 'Показание по тарифу №4', 'Житель', 'Источник'],
-        en: ['Reading date', 'Address', 'Unit', 'Service', 'Meter number', 'Place', 'Reading from tariff №1', 'Reading from tariff №2',
+        en: ['Reading date', 'Address', 'Unit', 'Account number', 'Service', 'Meter number', 'Place', 'Reading from tariff №1', 'Reading from tariff №2',
             'Reading from tariff №3', 'Reading from tariff №4', 'Contact', 'Source'],
```

Comment:
Can i18n translations be used server-side?

Reply:
it's better to move it to translations!